content/en/blog/2018/egress-tcp/index.md

```diff
@@ -349,7 +349,7 @@ becomes addressable by a local cluster domain name, for example by `mysqldb.vm.s
 entry to access this service; however, the service must be registered with Istio. To enable such integration, Istio
 components (_Envoy proxy_, _node-agent_, `_istio-agent_`) must be installed on the machine and the Istio control plane
 (_Pilot_, _Mixer_, _Citadel_) must be accessible from it. See the
-[Istio Mesh Expansion](/docs/examples/mesh-expansion/) instructions for more details.
+[Istio Mesh Expansion]/docs/tasks/virtual-machines/) instructions for more details.
```

Comment:
Should this say [Istio Mesh Expansion] or something else? [VM support] for example?